micro_mg3_0.F90

```diff
@@ -1081,11 +1115,24 @@ subroutine micro_mg_tend ( &
   real(r8) :: irad
   real(r8) :: ifrac
 
-  real(r8) :: nimey  !meyers ice nucleation
-  real(r8) :: niact(mgncol,nlev) ! dummy for modified activation
+  !Variables for accretion seeing autoconverted liquid    
+  real(r8) :: rtmp(mgncol,nlev) ! dummy for rain + autoconversion
+  real(r8) :: ctmp(mgncol,nlev) ! dummy for liq - autoconversion
+  real(r8) :: ntmp(mgncol,nlev) ! dummy for liq - autoconversion number
 
+  ! Variables for height calculation
+  real(r8) :: zhalf(mgncol,nlev)  !midpoint height
+  real(r8) :: ps,H
+      
   !cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
 
+  rtmp =0._r8  
+  ctmp =0._r8
+  ntmp =0._r8
```

Comment:
better to initialize these three arrays with other array variables below?

Reply:
and may need to put them into the `acc data create` directive as well.

Reply:
Done